### PR TITLE
Update main.py - gravity added

### PR DESCRIPTION
**Pull request by Haltie13**
Created adjustable gravity vector variable.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -9,6 +8,8 @@
 dt = 0
 
 player_pos = pygame.Vector2(screen.get_width() / 2, screen.get_height() / 2)
+player_velocity = pygame.Vector2(0, 0)
+gravity = pygame.Vector2(0, 500)  # Adjust gravity strength as needed
```
Comment (Taubeee) [inline main.py:12]: Consider adjusting parameters

File: main.py [hunk 2]
```diff
@@ -17,27 +18,36 @@
         if event.type == pygame.QUIT:
             running = False
 
-    # fill the screen with a color to wipe away anything from last frame
+    # fill the screen with a color to wipe away anything from the last frame
     screen.fill("purple")
 
-    pygame.draw.circle(screen, "red", player_pos, 40)
+    # Apply gravity
+    player_velocity += gravity * dt
```
Comment (Taubeee) [inline main.py:25]: LGTM

**Review by Taubeee — APPROVED** [review]
LGTM

File: main.py (hunk 1 above)
Comment (Haltie13) [reply main.py:12]: ok

File: main.py (hunk 2 above)
Comment (Haltie13) [reply main.py:25]: thx

**Comment by Haltie13** [comment]
Issue assigned

**Comment by Haltie13** [comment]
close #19

**Pull request merged**